muxpi/dut.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+echo "Setting up DUT" 
+stm_armv7 -clr
+stm_armv7 -print "DUT"
+stm_armv7 -dut
+echo "This script contains a lot of sleep commands as it needs to wait for the screens. Patience is key!"
+sleep 30s
+
+stty -F /dev/ttyACM0 115200 cs8 -cstopb -parenb
+cd /home/muxpi/
+#We need one screen to save the output of /dev/ttyACM0. Everything will be saved on 'serial-output'
+screen -d -m -S output 
+screen -S output -X stuff 'cat /dev/ttyACM0 > serial-output\n' 
+sleep 3s
+
+#This screen will be logging in the rpi and sending the commands to it. Note that when logging in, '\n' has to be passed multiple times as the login might fail. 
```

Comment:
wrap long lines

Reply:
Agreed. Lines should never be longer than 100 chars (preferably 80 chars according to me, but that's a matter of taste).